datalad/downloaders/configs/nda.cfg

```diff
@@ -9,6 +9,25 @@ authentication_type = nda-s3
 url = https://ndar.nih.gov/access.html
 type = nda-s3
 
+# Development instance
+[provider:NDA-development]
+url_re = https://development.nimhda.org($|/.*)
+credential = NDA-development
+# or some kind of html_form???
+authentication_type = http_basic_auth
+
+[credential:NDA-development]
+type = nda-s3
+
+[provider:NDA-development-s3]
+url_re = s3://TODO?
```

Comment:
Should this TODO remain?

Reply:
I think it is safe to remove all NDA-development* for now, will do, thanks